401d56/code-challenges/gradle/src/main/java/hashtable/Node.java

```diff
@@ -0,0 +1,45 @@
+package hashtable;
+
+public class Node {
+
+    private String key;
+    private String value;
+    private Node next;
+
+    public Node(String key, String value) {
+        this.key = key;
+        this.value = value;
+    }
+
+
+    public void setNext(Node node) {
+        this.next = node;
+    }
+
+    public void setKey(String key){
+        this.key = key;
+    }
+
+    public void setValue(String value) {
+        this.value = value;
+    }
+
+    public void Node(String key, String value){
```

Comment:
This is named the same as the class, but isn't the constructor. Why? myNode.Node(key, value) doesn't tell me what it's doing at all